Owncloud iOs Client/Network/Authentication/HandleCredentials.m

```diff
@@ -0,0 +1,41 @@
+//
```

Comment:
This file is not used in the app, XCode is even ignoring it. Please, remove it.

Reply:
👍